### PR TITLE
PEP 8 fixes to regression tests

### LINKED ISSUE #2504: NRS IFU regression tests not comparing to truth files
In the following tests:
```
tests_nightly/general/nirspec/test_nrs1_ifu_wcs.py
tests_nightly/general/nirspec/test_nrs1_ifu_wcs_2.py
tests_nightly/general/nirspec/test_nrs2_ifu_wcs.py
```
The comparison and the output WCS are the same, i.e. `imref` is never used, so the comparison of the output is being done with the output, not the truth file.

https://github.com/spacetelescope/jwst/blob/2b7967b8155dd09208d395515d42f7eb89e73f92/jwst/tests_nightly/general/nirspec/test_nrs1_ifu_wcs.py#L40-L44

So nothing is being tested.  This was caught by `flake8`.

Also, these 3 tests have the exact same format, so could benefit from being made into a single, parametrized test.

### PR DESCRIPTION
**Pull request by jdavies-st**
- Re-enable regression test `nirspec/test_spec2.py`
- Change `fits as pf` imports to the standard `from astropy.io import fits`
- Resolve #2504, fixing the NRS WCS tests, combining the IFU ones into a single parametrized test
- Make `miri/test_setpointing.py` output in the standard test outputs, not TEST_BIGDATA

### CODE REVIEW

File: jwst/tests_nightly/general/nirspec/test_nrs_ifu_wcs.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    truth_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-ifu', truth_file)
+
+    result = AssignWcsStep.call(input_file,
+                       save_results=True,
+                       suffix='assign_wcs'
+                       )
+    result.close()
+
+    im = ImageModel(result.meta.filename)
+    imref = ImageModel(truth_file)
```
Comment (nden) [inline jwst/tests_nightly/general/nirspec/test_nrs_ifu_wcs.py]: Strictly speaking ifu files should be opened with `IFUImageModel` but I think `ImageModel` will work in this case as well. 
Do you mind adding a for loop and checking a few more slices, for example [0, 9, 16, 23, 29].  This test runs only on slice 0. I guess it hasn't been updated for a very long time.

File: jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py [hunk 2]
```diff
@@ -21,35 +19,25 @@ def test_nirspec_nrs1_wcs(_bigdata):
...
-    output_file_base, output_file = add_suffix('nrs1_fs_wcs_output.fits', 'assignwcsstep')
-
-    try:
-        os.remove(output_file)
-    except:
-        pass
-
     input_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-fs', 'jw00023001001_01101_00001_NRS1_ramp_fit.fits')
-    ref_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-fs', 'jw00023001001_01101_00001_NRS1_assign_wcs.fits')
+    ref_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-fs', 'jw00023001001_01101_00001_NRS1_ramp_fit_assign_wcs.fits')
```
[the remark this answers is not in the document]
Comment (stscieisenhamer) [reply jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py]: Yes, but that should be being created in the local directory, and `ref_file` is pointing to the one in `bigdata`.

File: jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py [hunk 3]
```diff
@@ -21,35 +19,25 @@ def test_nirspec_nrs1_wcs(_bigdata):
...
 
-    AssignWcsStep.call(input_file,
-                       output_file=output_file_base, name='assignwcsstep'
+    result = AssignWcsStep.call(input_file,
```
Comment (stscieisenhamer) [inline jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py]: nitpick: did an IDE allow this indentation? Though allowed, not a standard. Either:
```
    AssignWcsStep.call(
        input_file,
        output_file=output_file_base,
        name='assignwcsstep'
    )
```
which is my preference, or
```
    AssignWcsStep.call(input_file,
                       output_file=output_file_base,
                       name='assignwcsstep')
```

File: jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nden) [reply jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py]: Ah, right.

File: jwst/tests_nightly/general/nirspec/test_nrs_ifu_wcs.py (hunk 1 above)
Comment (jdavies-st) [reply jwst/tests_nightly/general/nirspec/test_nrs_ifu_wcs.py]: Good to know.  If you'd like to make that change in your PR adding more slices, I'll leave that to you.

File: jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py (hunk 3 above)
Comment (jdavies-st) [reply jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py]: Yes, that is ugly.  Will fix.

Comment (philhodge) [reply jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py]: @stscieisenhamer That's not what PEP 8 says.  There are examples showing multiple arguments on each line when aligning after the '('.

**Comment by jdavies-st** [comment]
Resolves #2478.

**Pull request merged**